os/fs/tmpfs/Kconfig

```diff
@@ -69,5 +70,24 @@ config FS_TMPFS_FILE_FREEGUARD
 		little more memory than needed is always allocated.  This permits
 		the file to shrink without so many realloctions.
 
+config FS_TMPFS_USE_NHEAP
+	bool "TMPFS_USE_NHEAP"
+	default n
+	depends on MM_NHEAPS > 1
+	depends on MM_REGIONS > 1
```

Comment:
In order to use default heap index = 2, the minimum number of heaps and regions must be at least 3. So, I think this condition is not sufficient?

Reply:
I will modify default heap index = 1